pyaerocom/aeroval/modelmaps_helpers.py

```diff
@@ -1,20 +1,18 @@
+from typing import Any
+import itertools
 import cartopy.crs as ccrs
 import matplotlib
 import matplotlib.pyplot as plt
-from cartopy.mpl.geoaxes import GeoAxes
-from matplotlib.axes import Axes
 from matplotlib.colors import ListedColormap, to_hex
 from seaborn import color_palette
 import io
 import xarray
 import pandas as pd
 import glob
 import os
-
-try:
-    from geojsoncontour import contourf_to_geojson
-except ModuleNotFoundError:
-    contourf_to_geojson = None
+import numpy as np
+import geojson
```

Comment:
`geojson.geometry.DEFAULT_PRECISION = 5`

for compatibility with geojsoncontour https://github.com/bartromgens/geojsoncontour/blob/46fb7a5c4b7202a61fa6201518a1859bb600a382/geojsoncontour/contour.py#L14 .

(default in geojson is 6: https://pypi.org/project/geojson/#default-and-custom-precision)

Reply:
I set it to 3, as we don't need that much precision and it doing so decreases the size of the files